src/dex-ui/pages/dao/MultiSigDAODashboard/Settings.tsx

```diff
@@ -0,0 +1,233 @@
+                    </Text>
+                    <IconButton
+                      size="xs"
+                      variant="link"
+                      aria-label="copy-member-id"
+                      fontSize="17px"
+                      icon={<CopyIcon color={Color.Neutral._500} />}
+                    />
+                    {isAdmin ? (
+                      <Tag textStyle="p small medium" color={Color.Grey_Blue._500}>
+                        Admin
+                      </Tag>
+                    ) : (
+                      <></>
+                    )}
+                  </Flex>
+                  <Flex direction="row" gap="4" alignItems="center" height="20px">
+                    <IconButton
+                      size="xs"
+                      variant="link"
+                      aria-label="refresh-member"
+                      fontSize="17px"
+                      onClick={() => handleReplaceMemberClick(member.accountId)}
+                      icon={<RefreshIcon />}
+                    />
+                    {!isAdmin ? (
+                      <IconButton
+                        size="xs"
+                        onClick={() => handleDeleteMemberClick(member.accountId)}
+                        variant="link"
+                        aria-label="delete-member-id"
+                        fontSize="17px"
+                        icon={<DeleteIcon color={Color.Teal_01} />}
+                      />
+                    ) : undefined}
+                  </Flex>
+                </Flex>
+              );
+            })}
+          </SimpleGrid>
+        </DAOFormContainer>
+        <DAOFormContainer>
+          <Flex direction="column" gap={2} marginBottom="0.4rem">
+            <Text textStyle="p medium medium">Governance</Text>
+            <Text textStyle="p small regular" marginBottom="0.7rem" color={Color.Neutral._500}>
+              Manage the governance related DAO properties.
+            </Text>
+            <Divider />
+          </Flex>
+          <SimpleGrid minWidth="100%" columns={2} spacingX="2rem" spacingY="1.2rem">
+            <Flex
+              direction="column"
+              bg={Color.White_02}
+              justifyContent="space-between"
+              border={`1px solid ${Color.Neutral._200}`}
+              borderRadius="4px"
+              padding="1.5rem"
+            >
+              <Flex direction="row" justifyContent="space-between" alignItems="center">
+                <Flex direction="column" gap="1">
+                  <Text textStyle="p xsmall medium" color={Color.Neutral._500}>
+                    THRESHOLD
+                  </Text>
+                  <Text textStyle="p large medium">{`${threshold} / ${memberCount}`}</Text>
+                </Flex>
+                <Button
+                  type="button"
+                  variant="primary"
+                  padding="10px 10px"
+                  height="40px"
+                  onClick={() => handleChangeThresholdClick()}
```

Comment:
Can just be `onClick={handleChangeThresholdClick}`

Reply:
Done